languages/README.md

```diff
@@ -0,0 +1,165 @@
+definition of roles and responsibilities as well as guidelines that translators 
+and reviewers should follow.
+
+## Status of this document
+
+This is a working draft. Changes could be made anytime.
+
+## Table of Contents
+
+- [Roles](#roles)
+- [How to translate](#how-to-translate)
+- [How to add a new language](#how-to-add-a-new-language)
+- [How to publish translation tasks](#how-to-publish-translation-tasks)  
+- [Translation style guide](#translation-style-guide)
+
+## Roles
+
+There are two categories of roles:
+
+1. **Translator.** This is anyone who participates in the translation of any 
+   CloudEvents documents. One becomes a formal translator once a Pull Request (PR) 
+   of their translation is merged.
+
+2. **Reviewer.** A reviewer is responsible for checking new translations 
+   and answering related issues.
+   Typically, reviewers of each language are members of a Github team which is named 
+   after the language code(i.e., zh-CN reviewers).
+
+### Translators
+
+A translator can start translating by following [the translation instructions](#how-to-translate).
+Translators should follow the [translation style guide](#translation-style-guide).
+
+### Reviewers
+
+Anyone, who builds [a new language structure](#how-to-add-a-new-language), will be the first reviewer
+of that language.
+
+Reviewers should track translation state of each document by editing 
+`languages/{language-code}/translations.md` file.
+
+There are 4 types of status:
+- Ready to start: any document is ready to be translated stay in this state.
+- Started: indicating someone is translating this document
+- PR reviewing: a translator finished the translation, and the work is being reviewed.
+- PR merged: the translation is successfully merged into the spec.
+
+Initially, the last edit time will be the time when the translation is merged.
+Reviewers should ensure translations are up-to-date with the English version as it changes, meanwhile 
+update the last edit time as well. 
+
+The following rules govern adding and removing reviewers:
+
+- Reviewers can be proposed via a PR to edit the `languages/{language-code}/reviewer.md` file.
+
+- Reviewers can be removed via a PR to edit the `languages/{language-code}/reviewer.md` file.
+
+## How to add a new language 
+
+This section is mainly for anyone who needs to build a new language structure.
+
+Fork [cloudevents/spec](https://github.com/cloudevents/spec/tree/main)
+to your Github repo. 
+
+Then check the [language list](languages.md) to see if the language
+you'd like to translate into already exists. If it doesn't exist, you can start to add a new one.
+
+### Step 1. build the structure
+
+Create a directory under `languages` directory, named after the specific 
+[language code](http://www.lingoes.net/en/translator/langcode.htm).
+
+Add a new item to the [language list](languages.md), which linking to the directory 
+you just created. 
```

Comment:
I assume this means create a new PR just to edit this md file, right? We should say that, and have them do it immediately before they do any real work so they can stop someone else from starting in parallel

Reply:
> I assume this means create a new PR just to edit this md file, right? 

Yes. And I'll clearly describe how to do it here.